lib/ansible/plugins/connection/winrm.py

```diff
@@ -25,18 +25,23 @@
 import shlex
 import traceback
 import json
-import xmltodict
 
-from ansible.compat.six.moves.urllib.parse import urlunsplit
 
+from ansible.compat.six.moves.urllib.parse import urlunsplit
 from ansible.errors import AnsibleError, AnsibleConnectionFailure
+
 try:
     import winrm
     from winrm import Response
     from winrm.protocol import Protocol
 except ImportError:
     raise AnsibleError("winrm is not installed")
 
+try:
+    import xmltodict
+except ImportError:
+    raise AnsibleError("winrm is not installed")
+
```

Comment:
Misleading error message?  Should it say "xmltodict is not installed.  Needed to use winrm connection type"?